include/octo-logger-cpp/log.hpp

```diff
@@ -71,21 +78,19 @@ class Log
     }
 
     template <typename... T>
-    void formatted(fmt::format_string<T...> fmt, T&&... args)
+    Log& formatted(fmt::format_string<T...> fmt, T&&... args)
     {
         if (stream_)
         {
             *stream_ << fmt::format(fmt, std::forward<T>(args)...);
         }
+        return *this;
     }
 
-    template <typename... Args>
-    void formattedf(char const* fmt, Args... args)
+    Log& with_context(ContextInfo context_info)
```

Comment:
Please replace this with `Log error(std::string_view extra_identifier, ContextInfo context_info) const;` (and other log levels).

Reply:
reverted